inst/rmd/qc_report.rmd

```diff
@@ -20,7 +20,27 @@ output:
 library(SingleCellExperiment)
 ```
 
-# `r params$sample` Experiment Information
+# Processing Information for `r params$sample`
+
+```{r echo=FALSE}
+# extract sce metadata containing processing information as table
+processing_info <- sce@metadata %>%
```

Comment:
This probably works most of the time, but it is dangerous, as `sce` is not defined in this notebook.

I also switched to use the `metadata()` function, as I think this is a bit more standard than directly accessing the slot name. 

```suggestion
processing_info <- metadata(params$sce) %>%
```